cmd/access/main.go

```diff
@@ -206,14 +207,17 @@ func main() {
 			return nil
 		}).
 		Component("RPC engine", func(node *cmd.FlowNodeBuilder) (module.ReadyDoneAware, error) {
+			panicyBlocks := &accessstorage.Blocks{
```

Comment:
For each storage interface that is being used, create and use a wrapper that panics on error